phpstan.neon

```diff
@@ -1,5 +1,5 @@
 parameters:
-    level: 7
+    level: 4
```

Comment:
better create a baseline or drop phpstan at all as we are heading to psalm anyways 👍